### PR TITLE
Two lines footer

### PR DESCRIPTION
**Pull request by robert00s**
Close: #1933

New option that moves progress bar to separate line in footer.

### CODE REVIEW

File: frontend/apps/reader/modules/readerfooter.lua [hunk 1]
```diff
@@ -930,6 +943,23 @@ function ReaderFooter:addToMainMenu(menu_items)
...
+                text = _("Progress bar on separate line"),
+                checked_func = function()
+                    return self.settings.progress_bar_separate_line
+                end,
+                enabled_func = function()
+                    return not self.settings.disable_progress_bar
+                end,
+                callback = function()
+                    self.settings.progress_bar_separate_line = not self.settings.progress_bar_separate_line
+                    self:updateFooterContainer()
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readerfooter.lua]: Should these three lines be refactored into one update function of some sort?

**Comment by NiLuJe** [comment]
Speaking of the FW 4.17 progress bar: it's an overlay, so it doesn't directly affect the page layout (especially given the still somewhat generous hardcoded free footer space).

(It's obvious on a cover page, because it doesn't sit right at the bottom edge of the screen, so you get to see a few lines of the image *below* the bar ;)).

File: frontend/apps/reader/modules/readerfooter.lua (hunk 1 above)
Comment (robert00s) [reply frontend/apps/reader/modules/readerfooter.lua]: Done :)

**Comment by AlanSP1** [comment]
This looks great, thanks, just like I imagined.

**Pull request merged**